src/Item/Item.service.ts

```diff
@@ -423,7 +423,25 @@ export class ItemService {
     dbCollection: CollectionAttributes | undefined,
     eth_address: string
   ): Promise<FullItem> {
-    const canUpsert = await new Ownable(Item).canUpsert(item.id, eth_address)
+    let isManager = false
+    const isDbCollectionPublished =
+      dbCollection &&
+      (await this.collectionService.isDCLPublished(
+        dbCollection.contract_address!
+      ))
+
+    if (isDbCollectionPublished) {
+      const remoteCollection = await collectionAPI.fetchCollection(
+        dbCollection!.contract_address!
+      )
```

Comment:
There's a method (`getCollection`) that does this in the `Collection.service`, what do you think about using it?

Reply:
thanks for the feedback! sure! it'll look better and more readable